container/falcon-container-terraform/README.md

```diff
@@ -0,0 +1,71 @@
+# Terraform to demo Falcon Container Runtime Protection
+
+This terraform demo...
+ * Creates a single AKS cluster
+ * Creates a single Azure VM instance to manage the cluster
+ * Creates a Azure Container Registry (ACR)
+ * Enables Azure Key Vault
+ * Stores Falcon Credentials in the key vault
+ * Downloads the Falcon Container Sensor
+ * Pushes the Falcon Container Sensor to ACR
+ * Deploys the Falcon Container Sensor to the AKS Cluster
+ * Deploys the vulnerable.example.com application to AKS
+
+User then may
+ * Show that container workload (vulnerable.example.com) appears in Falcon Console (under Hosts, or Containers Dashboard)
+ * Visit vulnerable.example.com application and exploit it through the web interface
+ * Show detections in Falcon Console
+
+### Prerequsites
+ - Authenticated access to Azure
+ - Have Containers enabled in Falcon console (CWP subscription)
+ - Falcon CID and API key and secret with Sensor Download
+
+### Usage
+
+ - Open your Azure cloud shell
+ - Verify that your are on the correct subscription
+ - Verify that your identity in upper right corner is correct)
+ - Paste the following to your cloud shell
+ 
+```
+wget https://raw.githubusercontent.com//CrowdStrike/Cloud-Azure/master/container/falcon-container-terraform/demo
+```
+```
+/bin/bash demo up
+```
+
+NOTE:
+
+* This demo will prompt for the required API Client ID, Secret and CID for your falcon platform.
+* This demo will also prompt on whether to install the LUMOS (falcon-container) sensor
+    * If you choose not to install LUMOS the Kernel Mode Falcon-Node-Sensor will be deployed via Helm Chart
+
+### Tear Down
+
+NOTE: This demo will place you in a session to the `helper` VM which you must exit to tear down the environment.
+
+```
+cd ~/falcon-container-terraform && /bin/bash/demo down
```

Comment:
`/bin/bash/demo` seems bogus. Can you please double check?

Reply:
good catch